src/subdomains/supporting/bank/bank-account/bank-account.entity.ts

```diff
@@ -36,6 +36,9 @@ export class BankAccount extends IEntity implements BankAccountInfos {
   @Column({ length: 256 })
   iban: string;
 
+  @Column({ nullable: true })
+  synced: boolean;
```

Comment:
`default: false`?

Reply:
I think, we don't need it. The column is only temporary for the sync 👍